template/pdk.gen.go.ejs

```diff
@@ -110,17 +120,34 @@ func _<%= ex.name %>() int32 {
 		switch (v) {
 		<% schema.enum.forEach((variant) => { -%>
 		case <%= makePublic(variant) %>:
-			return "<%= variant %>"
+			return `"<%= variant %>"`
```

Comment:
so this isn't really a template thing, but when the input gets json decoded it has quotes on it?

Reply:
yea it's strange - not related to the template as far as I can tell.

The JSON on the way in doesn't seem wrong either. It's a little weird having to handle the enum this way since we have to kind of fake them in Go. maybe I shouldn't try to convert from/to strings but byte slices..

Reply:
@nilslice strings in json are encoded with `"`, if we pass in a json string, it will be like: `"banana"`. So `banana` is not valid json.

`xtp-typescript-bindgen` doesn't actually json decode the strings. It uses them as is. I pushed a commit that brings go-bindgen's behavior in line with typescript-bindgen. Feel free to undo the commit

Reply:
yea thank you! we actually have an improved implementation that doesn't try to force int enums into a string shaped hole 😎 on #5